src/main/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionary.java

```diff
@@ -83,7 +80,8 @@ protected Object doWork() {
         SAMFileWriter samWriter = null;
         //This writes the header with sequenceDictionary
         try {
-            samWriter = createSAMWriter(OUTPUT, REFERENCE_SEQUENCE, samHeader, false);
+            // do not use createSAMWriter because it does not create a plain text file without a .sam extension
+            samWriter = new SAMFileWriterFactory().makeSAMWriter(samHeader, false, OUTPUT);
```

Comment:
There's an existing test called `CreateSequenceDictionaryIntegrationTest.testBasic()` but it's grossly deficient and doesn't actually assert anything other than that it doesn't crash.  If you could update that to check that it produced the expected sequence dictionary that would be great.